src/parser.js

```diff
@@ -240,6 +236,7 @@ export function embedCode(kvMap, { filePath, originalPath, label }) {
  * @return {Array}
  */
 export function parse(content, baseDir, options = {}) {
+    const markdownLinkFormatRegExp = /\[([^\]]*?)\]\(([^\)]*?)\)/gm;
```

Comment:
```diff
- const markdownLinkFormatRegExp = /\[((?:[^\]]|\\.)*?)\]\(((?:[^\)]|\\.)*?)\)/gm; 
+ const markdownLinkFormatRegExp = /\[([^\]]*?)\]\(([^\)]*?)\)/gm;
```

This PR revert `markdownLinkFormatRegExp` of https://github.com/azu/gitbook-plugin-include-codeblock/commit/e1866d9a2842728aebeeadac76c8cc1fbbfad764#diff-6947033678b93d106e25614dd972e66fR13
All test case passed.

Is this correct?

/cc @lwchkg

Reply:
Reading... give me some time.

Reply:
Ok. Thanks for quick response :)

Reply:
The reverted expression does not handle the markdown escapes "\]" and "\)", so theoretically it's not complete.

But considering the bug is a freeze I'd say continue to commit.

It appears that we can't use "|" in JS RegExp because efficiency is too low. If JS RegExp does not allow any means to disable backtracking, I think we'll need to implement this without RegExp. Takes some time to implement anyway.